yaki_mobile/assets/translations/fr.json

```diff
@@ -103,7 +102,8 @@
   "IWorkthisMorning": "Je travaille ce",
   "IWorkthisAfternoon": "Je travaille cet",
   "Iam": "Je suis",
-  "office": "Au Bureau",
+  "OnSite": "Sur site",
```

Comment:
Was perfectly fine to keep "Au Bureau" just the trad key need will remain "onSite" (camelCase)